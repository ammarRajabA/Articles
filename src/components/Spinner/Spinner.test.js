import React from 'react'
import { shallow ,mount } from 'enzyme';

import Spinner from './Spinner.component';

describe('Spinner component',function () {
	it('match snapshot',function () {
		const component = mount(<Spinner />);
		expect(component).toMatchSnapshot();
	})
})