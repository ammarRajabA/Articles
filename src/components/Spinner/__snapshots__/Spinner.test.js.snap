// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component match snapshot 1`] = `
<Spinner>
  <div
    className="container"
  >
    <div
      className="loader"
    />
    <h1 />
  </div>
</Spinner>
`;
